advanced/accessibility.md

```diff
@@ -11,27 +11,36 @@ keywords: accessibility wai aria jaws nvda
+When navigating to a TinyMCE editor, the keyboard focus (cursor) will be in the content area. The up and down arrow keys navigate between the lines of the editor, such as within and between paragraphs, headings and other items such as links.
 
-Press `Alt+F9` to move focus to the menu bar. `Left Arrow` and `Right Arrow` navigates through the top menu items. `Down Arrow` opens the menu. Press `Esc` once to go back to the top level. `Right Arrow` on a submenu navigate into them while `Esc` backs out.
+### Navigating the menu bar
 
-Alternatively, press `Alt+F10` to focus on the toolbars. The first toolbar has focus and you can `Tab` or arrow through the items<!-- DEV should items be buttons -->. Submenu items and panels are opened with `Down Arrow` and closed with `Esc`. When you reach the end of one toolbar the `Right Arrow` and `Left Arrow` wrap to the next and previous toolbars respectively. `Esc` brings you back into the central content area.
+To focus on the editor menu bar, press **Alt+F9**. The **Left Arrow** and **Right Arrow** keys navigate through the top menu items. The **Down Arrow** opens the highlighted menu and moves focus to the first menu item. To activate or select the highlighted menu item, use the **Enter**, **Return**, or **Spacebar**. Press the **Esc** key to collapse the menu. The **Right Arrow** key opens submenus and the **Esc** key collapses submenus. To return focus to the content area from the menu bar, use the **Esc** key.
 
-Dialogs such as Insert/Edit or Image/Video are opened from either the menus or the toolbars. These are multi-page dialogs with at least two tabs at the top. The `Right Arrow` and `Left Arrow` moves between the tabs in the tab list. `Space` makes the current tab active. Tab moves into the panel and through the controls. `Save` accepts changes, `Esc` or `Cancel` discards changes.
+### Navigating the toolbar
 
-You can insert emoticons or change the text and background colors on the toolbars and in the formatting menu. These are panels you open by pressing `Down Arrow` or `Enter` on their respective menu/toolbar items and then use the arrow keys to select the item you want to be applied/inserted. All of these items are screen reader enabled allowing the visually impaired to navigate the toolbars and menu structure.
+#### General toolbar navigation
 
+To focus on the editor toolbars, press **Alt+F10**; which moves the keyboard focus to the first toolbar button on the first toolbar. To move between toolbar buttons _within_ toolbar groups (visual groups of toolbar buttons), use the **Left Arrow** and **Right Arrow** keys. To move the focus _between_ toolbar groups, use the **Tab** or **Shift+Tab** keys. When you reach the end a toolbar group the **Right Arrow** key will return the focus to the start of the toolbar group. To activate or select the highlighted toolbar button, use the **Enter**, **Return**, or **Spacebar**. To return focus to the content area from the menu bar, use the **Esc** key.
```

Comment:
"reach the end a toolbar group" missing "of" here.

Reply:
Also is missing "keys" again after "Enter, Return, or Spacebar" (same issue as the menu bar).

Reply:
"To return focus to the content area from the menu bar" pretty sure this is a copy/paste fail :stuck_out_tongue: as it should say toolbar.

Reply:
:facepalm:

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1834/commits/9179e84030d4f64333b2f0862f175f55b312c9a6